bot/skills/__init__.py

```diff
@@ -34,16 +38,61 @@
 logger = logging.getLogger(__name__)
 
 
+class ChatCommandHandler(CommandHandler):
+    """ChatCommandHandler is class-wrapper for `CommandHandler`. It provides default `chat_id` filtering.
+    `chat_id` takes from `config.get_group_chat_id() -> str` and it could be either
+    chat_username :: str (for public groups) or chat_id :: int (for private or public). So, this wrapper
+    support both type of `chat_id`, adds chat_id filtering and set `run_async` to True by default.
+    """
+
+    def __init__(
+        self,
+        command: SLT[str],
+        callback: Callable[[Update, CCT], RT],
+        filters: BaseFilter = None,
+        allow_edited: bool = None,
+        pass_args: bool = False,
+        pass_update_queue: bool = False,
+        pass_job_queue: bool = False,
+        pass_user_data: bool = False,
+        pass_chat_data: bool = False,
+        run_async: Union[bool, DefaultValue] = DEFAULT_FALSE,
+    ):
+
+        # chat_id filtering: accept messages only from particular chat
+        chat_id_or_name = get_group_chat_id()
+        try:
+            f = Filters.chat(chat_id=int(chat_id_or_name))
+        except ValueError:
+            f = Filters.chat(username=chat_id_or_name.strip("@"))
+        filters = f if filters is None else filters & f
+
+        # run commands async by default
+        if run_async == DEFAULT_FALSE:
```

Comment:
I don't get it. What does `DEFAULT_FALSE` mean if it sets `run_async` to `True`?

Reply:
Here `DEFAULT_FALSE` allow me to know, that `run_async` argument comms with _default_ value. Only in this particular case, I'd like to set it to `True`.

`DefaultValue` is a generic from `python-telegram-bot` library: `telegram.utils.helpers.DefaultValue`. They describe it like so:

```
Wrapper for immutable default arguments that allows to check, if the default value was set
    explicitly. Usage::

        DefaultOne = DefaultValue(1)
        def f(arg=DefaultOne):
            if arg is DefaultOne:
                print('`arg` is the default')
                arg = arg.value
            else:
                print('`arg` was set explicitly')
            print(f'`arg` = {str(arg)}')
...
```

In different words, I wanna change `run_async` only when nobody sets it from outside.

Reply:
How this is different from setting default value to True?

Reply:
I believe it was done to giving to parent class possibility to determinate than the `DefaultValue` was overloaded by some successors. 🤷

Reply:
> I believe it was done to giving to parent class possibility to determinate than the `DefaultValue` was overloaded by some successors. 🤷 

Heresy

Reply:
Zwar